styles/components/grid.css

```diff
@@ -0,0 +1,21 @@
+.grid-container-2{
+    max-width: 800px;
+    border:1px solid var(--border-color)
+    
+}
+.grid-container-3{
+    max-width: 800px;
+    border:1px solid var(--border-color)
+    
+}
+.row{
+    display: flex;
```

Comment:
Use  _display : grid_ instead of _flex_ for the container as it says grid-container, or else rename it to flex-container if you want to use flex.